### ISSUE TITLE
Cannot type, even with LANG=en_US.UTF-8

### ISSUE THREAD

**Issue by chaosmonk1** [profanity-im/profanity#1233]
## Expected Behavior
According to the [FAQ](http://www.profanity.im/faq.html?LMCL=NvFcwg#utf8encoding), profanity should accept input when `LANG` is set to `en_US.UTF-8`.

## Current Behavior
profanity does not accept input, even when all locale variables are set to `en_US.UTF-8`

## Possible Solution
There are numerous issues which report being unable to type, but as far as I can see, the reported solution is always to set `LANG` or some other locale variable to en_US.UTF-8.  However, all of my locale variables are set to en_US.UTF-8 already.  Are there other known causes for this symptom that I have overlooked in reading though the other threads?  

## Steps to Reproduce (for bugs)
1. Verify that locale variables are set to en_US.UTF8
```
$ locale
LANG=en_US.UTF-8
LANGUAGE=en_US:en
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC=en_US.UTF-8
LC_TIME=en_US.UTF-8
LC_COLLATE="en_US.UTF-8"
LC_MONETARY=en_US.UTF-8
LC_MESSAGES="en_US.UTF-8"
LC_PAPER=en_US.UTF-8
LC_NAME=en_US.UTF-8
LC_ADDRESS=en_US.UTF-8
LC_TELEPHONE=en_US.UTF-8
LC_MEASUREMENT=en_US.UTF-8
LC_IDENTIFICATION=en_US.UTF-8
LC_ALL=
```
2. Run profanity
```
$ profanity # cannot type
```
3. Also set `LC_ALL` to `en_US.UTF-8` to be safe
```
$ export LC_ALL=en_US.UTF-8
```
4. Run profanity
```
$ profanity # still cannot type
```

## Environment
* Give us the version and build information output generated by `profanity -v`
```
$ profanity -v
Profanity, version 0.7.1
Copyright (C) 2012 - 2019 James Booth <boothj5web@gmail.com>.
Copyright (C) 2019 Michael Vetter <jubalh@iodoru.org>.
License GPLv3+: GNU GPL version 3 or later <https://www.gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Build information:
XMPP library: libmesode
Desktop notification support: Enabled
OTR support: Enabled (libotr 4.1.1)
PGP support: Enabled (libgpgme 1.6.0)
OMEMO support: Disabled
C plugins: Enabled
Python plugins: Enabled (2.7.12)
GTK icons: Enabled
```
* Operating System/Distribution
I am running Trisquel 8, which is based on Ubuntu 16.04.  I compiled profanity 7.1 and libmesode 9.3 from source, and installed all other dependencies from my distro's repositories.  I did not modify `configure.ac`.
* glib version
```
$ apt policy libglib2.0-dev | grep Installed
  Installed: 2.48.2-0ubuntu4.4
```

Thanks.

**Comment by jubalh** [comment]
I have never heard about such an issue.

I have:
```
locale
LANG=en_US.UTF-8
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_PAPER="en_US.UTF-8"
LC_NAME="en_US.UTF-8"
LC_ADDRESS="en_US.UTF-8"
LC_TELEPHONE="en_US.UTF-8"
LC_MEASUREMENT="en_US.UTF-8"
LC_IDENTIFICATION="en_US.UTF-8"
LC_ALL=
```

on openSUSE Tumbleweed. It all works fine. Does this also happen if you run `profanity` from the Trisquel repo? As it's a Debian derivate I suppose they have it there too, just in an older version?

**Comment by chaosmonk1** [comment]
> Does this also happen if you run `profanity` from the Trisquel repo?
> As it's a Debian derivate I suppose they have it there too, just in an
> older version?

Thanks for your response.  Yes, Trisquel has an older version, 4.7.0.  I
just installed it and initially experienced the same behavior.  However,
I then noticed that I have some profanity configuration files in `~/.config`.
I deleted these, and now profanity works, so I suspect that there was an
error in one of the configuration files.  Perhaps I tried out profanity
once a long time ago, attempted to configure it, and then forgot that I
had done so.  I apologize for the noise.  Closing.
